EventProc/python/hcalVeto.py

```diff
@@ -5,3 +5,4 @@
 hcalVeto.parameters['pe_threshold'] = 5.0
 hcalVeto.parameters['max_time'] = 50.0
 hcalVeto.parameters['max_depth'] = 4000.0
+hcalVeto.parameters['min_pe'] = 1.
```

Comment:
maybe call this something like "back_min_pe"?  just so it's clear that this is only for back HCal?
Otherwise looks good for me

Reply:
Made the change.